### ISSUE TITLE
Radio group options name not updating

### ISSUE THREAD

**Issue by craigcooper11** [Draggable/formeo#275]
https://draggable.github.io/formeo/

Steps:

Drag and drop a Radio Group input. Change the name of the radio group options and click save, then render form. The rendered radio group does not reflect the name change.

![image](https://user-images.githubusercontent.com/37030956/84063990-88431c00-a987-11ea-88e0-f477fbc3ae10.png)

**Comment by zaydons** [comment]
I was also able to replicate this, might end up digging through the source later to see if I could submit a PR

**Comment by bsappey** [comment]
@zaydons did you have any luck with this? I am still seeing the same issue in the current build.

**Comment by dougauerbach** [comment]
I agree, I'm getting customers complaining about this. Definitely a bug. The workaround is to enter the settings for each option, rather than just typing into the label.
